profile/README.md

```diff
@@ -0,0 +1,145 @@
+# OpenShift Service Mesh
+
+Repositories for building OpenShift Service Mesh (OSSM) for the [OpenShift
+Container Platform][ocp].
+
+> **Note**: For general information about service mesh concepts click [here].
+
+> **Warning**: This organization deals *only* with builds of the 3.x series of
+> OSSM.
+
+[here]:https://www.redhat.com/en/topics/microservices/what-is-a-service-mesh
+[ocp]:https://www.redhat.com/en/technologies/cloud-computing/openshift/container-platform
+
+## About
+
+OpenShift Service Mesh (OSSM) is based on [Istio] and tries to follow that
+upstream project very closely.
+
+The repositories you find here will generally be up to date with the upstream
+Istio repositories, with some small changes mainly related to building and
+shipping releases. In the repositories section we'll provide a brief overview of
+some of the repositories in this org to help you get started.
+
+> **Note**: Ultimately **this organization is meant for OSSM developers, not
+> users**. If you're a user looking to install and use OSSM, see the [OpenShift
+> Container Platform Documentation][ocp-docs] for your version instead.
+
+[Istio]:https://github.com/istio
+[ossm-istio]:https://github.com/openshift-service-mesh/istio
+[ocp-docs]:https://docs.redhat.com/en/documentation/openshift_container_platform/
+
+## Development
+
+The vast majority of development of OpenShift Service Mesh (OSSM) happens in
+[upstream Istio], the exception being some areas related to [Envoy Proxy] or
```

Comment:
may be nice to add a short explanation of how istio relates to envoy before going into more details

Reply:
Sounds good, I did mention this as the first point in the `## About` section, what do you think should I change that section? Or do I need to just expand further here?

Reply:
I think just saying "Istio uses envoy as ..." in the `About` section or here in the beginning

Reply:
Sounds good, lmkwyt:

```suggestion
[upstream Istio], the exception being some areas related to [Envoy Proxy] (which Istio uses) or
```

Reply:
maybe add this link https://istio.io/latest/docs/ops/deployment/architecture/ or copy a sentence from there saying "The Istio data plane is composed of a set of intelligent proxies ([Envoy](https://www.envoyproxy.io/)) deployed as sidecars." ?

Reply:
Sounds good, I pushed e2b4e2a453d7c5d98ab48b28492a0d537a4dbbf2 with changes, LMKWYT?

Reply:
perrrrfect!